tests/e2e/creation_test.go

```diff
@@ -353,6 +360,46 @@ func (tc *testContext) validateDSC() error {
 	return nil
 }
 
+func (tc *testContext) testTrustedCABundle(t *testing.T) error {
+	CAConfigMapName := "odh-trusted-ca-bundle"
+	CADataFieldName := "odh-ca-bundle.crt"
+
+	if tc.testDSCI.Spec.TrustedCABundle.ManagementState == operatorv1.Managed {
+		foundConfigMap := &corev1.ConfigMap{}
+		err := tc.customClient.Get(tc.ctx, client.ObjectKey{
+			Name:      CAConfigMapName,
+			Namespace: tc.testDSCI.Spec.ApplicationsNamespace,
+		}, foundConfigMap)
+
+		if err != nil {
+			return fmt.Errorf("Config map not found, %w", err)
+		}
+
+		checkNewline := strings.HasSuffix(foundConfigMap.Data[CADataFieldName], "\n")
+
+		if checkNewline == false {
+			t.Log("Newline not found at the end of configmap")
+		}
```

Comment:
So, what is the point of the message and the check if does not fail test?
If it's needed to test scenarios with and without "\n" there must be separate tests.

Reply:
This is one of the questions I had [here](https://github.com/opendatahub-io/opendatahub-operator/pull/1346#discussion_r1873257790) also. If the test isn't going to pass or fail based on the check, then the check doesn't seem useful.

I think this log line and the check for the newline could be dropped altogether.

Reply:
> This is one of the questions I had [here](https://github.com/opendatahub-io/opendatahub-operator/pull/1346#discussion_r1873257790) also. If the test isn't going to pass or fail based on the check, then the check doesn't seem useful.

Oh, that's why I prefer rebasing in the same PR instead of recreation. To not to lose conversations.

> 
> I think this log line and the check for the newline could be dropped altogether.

+1